call/bitrate_allocator.cc

```diff
@@ -34,7 +34,7 @@ using bitrate_allocator_impl::AllocatableTrack;
 // Allow packets to be transmitted in up to 2 times max video bitrate if the
 // bandwidth estimate allows it.
 const uint8_t kTransmissionMaxBitrateMultiplier = 2;
-const int kDefaultBitrateBps = 300000;
+const int kDefaultBitrateBps = 100000;
```

Comment:
Can we make the value in api/transport/bitrate_settings.h a public value and then include there here instead of having two copies of the same value?